Pra.DbDisconnected.WPF/MainWindow.xaml.cs

```diff
@@ -15,222 +15,222 @@ public MainWindow()
         {
             InitializeComponent();
         }
-        DataSet dsBoekenLijst = new DataSet("Bibliotheek");
-        string XMLMap = Directory.GetCurrentDirectory() + "/XMLBestanden";
-        string XMLBestand = Directory.GetCurrentDirectory() + "/XMLBestanden/boeken.xml";
+        DataSet dsBookList = new DataSet("Bibliotheek");
+        string XMLDirectory = Directory.GetCurrentDirectory() + "/XMLBestanden";
+        string XMLFile = Directory.GetCurrentDirectory() + "/XMLBestanden/boeken.xml";
         private void Window_Loaded(object sender, RoutedEventArgs e)
         {
-            if (!LeesXML())
+            if (!ReadXML())
             {
-                MaakTabellen();
-                VulTabellen();
+                CreateTables();
+                FillTables();
             }
-            dgAuteur.ItemsSource = dsBoekenLijst.Tables[0].DefaultView;
-            dgBoek.ItemsSource = dsBoekenLijst.Tables[2].DefaultView;
-            BronGegevensAanpassen();
+            dgAuthors.ItemsSource = dsBookList.Tables[0].DefaultView;
+            dgBooks.ItemsSource = dsBookList.Tables[2].DefaultView;
+            SourceDataChange();
```

Comment:
Nu we toch dingen aan het hernoemen zijn, zou ik overwegen om deze specifieker te maken, bv. `UpdateComboBoxes`.